### DISCUSSION TITLE
Github education not processing my application, stuck in loop

### DISCUSSION THREAD

**Discussion by yannickdv01** [community/community#129312] [GitHub Education]
### Hi everyone,

Im trying to upload my application, however on the 'addiotional info' page whenever i try to upload my proof and click submit a progress bar should appear. But in my case i get redirected to the same page again

**Comment by javawolfpack** [comment]
Same, and all my contacts at GitHub Education have sadly left so I have no way to get support except here where I haven't gotten a response now in days of waiting.
